partest-suite/src/test/resources/scala/tools/partest/scalajs/2.11.0/neg/t6446-additional.check

```diff
@@ -11,21 +11,23 @@ superaccessors   8  add super accessors in traits and nested classes
     extmethods   9  add extension methods for inline classes
```

Comment:
[Conveniently jump forward to the last updated .check files to skip the noise](#diff-91719e27cf3802a8f2167e22dbea938d)